src/stratis_cli/_error_reporting.py

```diff
@@ -223,6 +224,12 @@ def _interpret_errors_1(
             f"of the command that you requested: {error}"
         )
 
+    if isinstance(error, StratisCliSynthUeventError):
+        return (
+            f"stratisd reported an error in generating a synethetic "
```

Comment:
Maybe this would be better as something like "stratis encountered"? The daemon isn't involved in any of this so it could be confusing for users if the error mentions stratisd.

Reply:
I changed it to "stratis reported", via a force-push of an amended commit.